src/group/group.model.ts

```diff
@@ -77,6 +77,6 @@ const groupSchema = new Schema({
   },
 });
 
-groupSchema.virtual("childrens", { ref: "group", localField: "children", foreignField: "kartoffelID" });
+groupSchema.virtual("childrenGroup", { ref: "group", localField: "children", foreignField: "kartoffelID" });
```

Comment:
rename childrenGroup to populatedChildren
Also, add comment about what this line does.

Reply:
done